fhir/resource_structure_dev/research_study.json

```diff
@@ -0,0 +1,111 @@
+{
+  "resourceType" : "ResearchStudy",
+  "id" : "example-study-ccpuk",
+  "url" : "",
+  "name" : "CCPUK",
+  "title" : "ISARIC/WHO Clinical Characterisation Protocol UK",
+  "label" : [{
+    "type": {
+      "text": "Study Code"
+    },
+    "value": "CVCCPUK"
+  }],
```

Comment:
'relatedArtifact' could be used to store the original case report form for the study. Although the CRF could be site-specific within a larger study so may need to go elsewhere